Auth0/CredentialsManager.swift

```diff
@@ -40,14 +40,25 @@ public struct CredentialsManager {
         self.authentication = authentication
     }
 
-    /// Enable TouchID Authentication for additional securtity during credentials retrieval.
+    /// Enable Face ID / Touch ID Authentication for additional securtity during credentials retrieval.
     ///
     /// - Parameters:
     ///   - title: main message to display in TouchID prompt
     ///   - cancelTitle: cancel message to display in TouchID prompt (iOS 10+)
     ///   - fallbackTitle: fallback message to display in TouchID prompt after a failed match
+    @available(*, deprecated, message: "see enableBioAuth(withTitle title:, cancelTitle:, fallbackTitle:)")
```

Comment:
so just deprecated because of the name change? No difference in the parameters??

Reply:
Yes, for the sake of clarity it should be the more generic `bio` as `touchid` might imply to some developers that it is ONLY for `touch`

Reply:
Thoughts on this?

Reply:
The new name is better but deprecating the method just because of the name doesn't sound right to me. Make this call 👍

Reply:
Additional comment: Message says "see enableBioAuth" but the method is called "enableBiometricAuth". Also if these methods are going to have the same logic, make one call the other instead of copy-pasting the logic.

Reply:
Made that change in the most recent commit.